metabase/metabase
# Cleanup JWT settings in Admin

Epic https://github.com/metabase/metabase/issues/19428

Removes the toggle from the JWT settings form and adds it to the auth card in `Authentication` tab.

How to test:
- Run Metabase EE
- Go to Admin -> Settings -> Authentication -> JWT
- Enter `https://example.test` for the url and click `Generate key`
- Click `Save and enable`
- Go back to `Authentication` and make sure that the toggle is now visible and can be used to enable and disable jwt
- Try the same with `Save but don't enable` button

<img width="538" alt="Screenshot 2022-10-21 at 18 45 28" src="https://user-images.githubusercontent.com/8542534/197236557-70618a7e-cb57-4034-b5ec-37b6da6cb830.png">
